### ISSUE TITLE
What if the content is no longer there?

### ISSUE THREAD

**Issue by mrjoelkemp** [mrjoelkemp/linklink#21]
News sites have front pages that change all of the time. 

Keep a cached version of the page? 
##

**Comment by dandv** [comment]
See #48. Saving a snapshot of the page transforms linklink into a [web page archiving service](http://wiki.dandascalescu.com/reviews/online_services/web_page_archiving).

**Comment by mrjoelkemp** [comment]
Agreed on this blowing up the scope of linklink. However, even the selector generation assumes that the linked element is somewhere on the page. If a week old element is no longer on the home page, then the linklink is dead.

I'd be happy to use point to an archive service if the element could not be found on the live site. The user would be shown a message saying that the element is no longer on the page, so they're being taken to an archive. That's not a great UX, but avoids having to track linklinks and redirect them to the archive on the fly.

I'll look into the services on the link that you provided and see which one I might be able to redirect to.
